### DISCUSSION TITLE
error when trying to create the FarmVibes.AI cluster

### DISCUSSION THREAD

**Discussion by romtarshish22** [microsoft/farmvibes-ai#24] [Q&A] [answered]
hi,

when i try to run the follow:

bash farmvibes-ai.sh setup

im getting this:

farmvibes-ai.sh: line 3: $'\r': command not found
farmvibes-ai.sh: line 5: $'\r': command not found
: No such file or directorycli
farmvibes-ai.sh: line 7: $'\r': command not found
: No such file or directorycli
: No such file or directorycli
: No such file or directory/cli
farmvibes-ai.sh: line 11: $'\r': command not found
: No such file or directory/cli
farmvibes-ai.sh: line 13: $'\r': command not found
: No such file or directory/cli
farmvibes-ai.sh: line 15: $'\r': command not found
farmvibes-ai.sh: line 20: syntax error near unexpected token `$'{\r''
'armvibes-ai.sh: line 20: `main() {

*******************
i have tried run it on either wsl, ubuntu and PowerShell. 

please advise

thanks

**Comment by brsilvarec** [comment]
Hi @romtarshish22, 

 Thanks for the question. I do not believe our platform supports PowerShell installation. Can you post the output of install script on Ubuntu platform?

Can you also provide the output of this command `lsb_release -a`? (see the example)

<img width="221" alt="image" src="https://user-images.githubusercontent.com/7769154/206713226-c3745140-00c0-4876-b1be-c5d4ca04cf9f.png">

**Comment by romtarshish22** [comment]
Hi @brsilvarec , thank you for your answer. as i mentioned, i have tried also Ubuntu. here are the outputs:

here is the output in Ubuntu for the bash ./resources/vm/setup_farmvibes_ai_vm.sh command:
![image](https://user-images.githubusercontent.com/105434152/209914500-b5861b54-6117-40a2-b635-27d738c737b2.png)

here is the output for the  bash farmvibes-ai.sh setup command:
![image](https://user-images.githubusercontent.com/105434152/209914615-7c84d0b3-88d7-467b-9c98-e35d65ae30f4.png)


here is the output for the lsb_release -a command:

![image](https://user-images.githubusercontent.com/105434152/209914231-fb2786bd-e890-47a1-a370-1a1cc8a77e08.png)

**Reply by renatolfc** [reply]
I noticed you cloned your repo in the C:\ drive. Was that checked out with git for windows?

Can you please do a new clone in the native WSL filesystem?

**Comment by renatolfc** [comment, marked as answer]
No bash script will run correctly when cloning from Windows and running on a Linux distribution because of the way the OSs treat line endings.

For details and instructions on how to refresh line endings after a clone, please refer to https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings?platform=windows
